### ISSUE TITLE
Bitstream.cpp Fatal Out of range write

### ISSUE THREAD

**Issue by rasteron** [GarageGames/Torque3D#1487]
Hi, 

I'm still getting this type of fatal error when doing some connection benchmarks (2 - 4 tries), particularly using Linux and Windows:

`/t3d/Engine/source/core/stream/bitStream.cpp(234,0): {Fatal} - Out of range write`

with the latest v3.8 release source on both. 
 
Old Reference:
http://www.garagegames.com/community/forums/viewthread/5180/2#comment-849030

**Comment by rasteron** [comment]
To follow-up, here's a screenshot with LAN connection (Ubuntu 14.04 32-bit)

![screenshot](https://cloud.githubusercontent.com/assets/3676827/11915849/c9631622-a6f3-11e5-8110-aa96dd2fabc5.jpg)

**Comment by Azaezel** [comment]
That's generally a report on the equivalent of a memory leak for the networking, like say, mismatched pacupdate and unpackupdate in a given class. What were you testing in particular?

**Comment by rasteron** [comment]
Ah I see. I'm just trying to log in and out and doing some stress testing to see if my connection with the server is consistent enough.

**Comment by rasteron** [comment]
Is this normal or is there a workaround for this particular issue?

**Comment by Azaezel** [comment]
Despite the thread, that's not an issue. it's an issue *report* telling you something's gone pearshaped with networking your end. Would need additional data to drill down to what in particular is causing it.

**Comment by rasteron** [comment]
Ok thanks. I'm using the source release T3D 3.8, Windows 7 64bit for client (VS2008) and Ubuntu 14.04 LTS 32 bit (Virtualbox) and a separate installation. I have not made any changes with the mission level, engine source or scripts.

**Comment by Areloch** [comment]
You said you were logging in and out in order to stress test.

Does this happen every time? Does it happen around the same number of in/out iterations?

**Comment by rasteron** [comment]
@Areloch Yes it does and around the 3rd or 4th attempt.

**Comment by Areloch** [comment]
so consistently, 3rd or 4th attempt. Same level I'm assuming?

**Comment by rasteron** [comment]
That's right. Here's another one that I'm getting recently with the loading/connecting screen only at 50%, a timeout error at the end and it crashes on 3rd reconnect attempt. Info/debug color message and I guess this is a separate issue.. 

`ParticleData((null)) times[2] < times[1]`
